paper.md

```diff
@@ -82,254 +82,113 @@ testing and scientific study validations.
-them. For example, GHA workflows can be the bridge between HPC and 
-cloud infrastructures. The roadmap of Popper envisions executing 
-seamlessly on Kubernetes by handling all the logic of deploying a 
-scheduler under the hood, or connecting the application to an existing 
-Kubernetes-hosted batch scheduler. This means that from the point of 
-view of a user, they can run the same workflow regardless of the 
-backend infrastructure.
+### SLURM
 
-### **User friendliness**
+Here we talk about slurm
 
-An extension of the above; container-based workflow language such as 
-GHA hides complexities and runtime-specific setups of a container 
-runtime. The researcher or developer sees `Dockerfile` and
-`.workflow` files; system administrators configure the runtime and 
-provide a configuration file that users pass to Popper (e.g. GPU, 
-batch schedulers, etc.).
+### Kubernetes
 
-## Challenges
+Here we talk about Kubernetes
+
+### CI Services
+
+Here we talk about some CI services
 
-### **Technological barriers**
+## Execution Scenarios
 
-Software containers, being a relatively new technology, can be seen a 
-significant time investment given its associated learning curve.
+Here we describe the 3 different execution scenarios
 
-### **Cultural barriers**
+### **Single-Node local workflow execution**
 
-Carrying out experimentation or complex application testing by writing 
-workflows is not something practitioners are used to do; following 
-this approach requires a paradigm shift.
+Single node workflow execution in the local machine
+for development purposes
 
-### **Security and stability concerns**
+### **Workflow execution in the Cloud using Kubernetes**
 
-Container technology is currently moving relatively fast, with new 
-versions of container runtimes being released every week. In some 
-contexts this might be a deal breaker, as this represents a burden 
-from the point of view of system administrators.
+Escalating the workflow execution to a GPU enabled cluster in
+the cloud for production grade execution
 
-# Related Work {#sec:related}
+### **Exascale workflow execution in SLURM clusters**
 
-The problem of implementing multi-container workflows as described in 
-@Sec:intro is addressed by existing tools. We briefly survey some of 
-these technologies and compare them with Popper 2.0 by grouping them 
-in categories.
+Planetscale workflow execution is super computing environments
 
-### **Workflow languages**
+# Results {#sec:result}
 
-Workflow languages such as CommonWL [@amstutz_common_2016] and Yadage 
-[^yadage] allow users to specify workflows using a standard, 
-runtime-agnostic language that can be implemented by an engine. These 
-languages do not assume a containerized environment, although they 
-support container-based workflows. Compared to the GHA language, their 
-syntax is more complex and they give users more control over execution 
-aspects (e.g. specify scheduling policies).
+## Complexity
```

Comment:
can you expand what do you mean by complexity? I like the idea of quantifying complexity, what did you have in mind?